shop/.dockerignore

```diff
@@ -9,3 +9,6 @@
 !composer.*
 !yarn.*
 !.stylelint*
+!tests/
+!setup_test
```

Comment:
`vendor/` is created / filled by `composer` and is a standard naming schema. Technically it is created in the concrete shop implementation (via ONBUILD -> init.d/*php*.sh) - so could be overwritten by consumers of this base image, yes.